ddtrace/internal/telemetry/writer.py

```diff
@@ -251,7 +251,7 @@ def enable(self, start_worker_thread=True):
         return config._telemetry_metrics_enabled and super(TelemetryLogsMetricsWriter, self).enable(start_worker_thread)
 
     def add_log(self, level, message, stack_trace="", tags={}):
-        # type: (str, str, str, MetricTagType) -> None
+        # type: (str, str, str, Dict) -> None
```

Comment:
Does this still need to be a dict?

Reply:
That was a mistake lol. MetricTagType is a type of Dict, that's why this did not raise an typing error. I will revert this